lib/hal/impression/rpihal.ex

```diff
@@ -9,6 +9,7 @@ defmodule Inky.Impression.RpiHAL do
 
   @behaviour Inky.HAL
 
+  @color_map %{black: 0, white: 1, green: 2, blue: 3, red: 4, yellow: 5, orange: 6, miss: 1}
```

Comment:
I doubt that this is correct. For now, I've only tested black. The color map is eventually used by `PixelUtil.do_pixels_to_bits`, but I haven't figured out exactly what it's doing when it generates the bits.